x12-edi/2.5/modules/ROOT/pages/x12-edi-connector-hipaa.adoc

```diff
@@ -3,19 +3,19 @@
 
 Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Premium]
 
-Anypoint Connector for X12 EDI Connector 2.5 - HIPAA provides support for the Health Insurance Portability and Accountability Act (HIPAA) standards based on X12 Connector standards and version `005010`.
-Refer to this information for using X12 Connector for HIPAA transaction sets, and for how usage differs from the standard X12.
+Anypoint Connector for X12 EDI Connector 2.5 - HIPAA provides support for the Health Insurance Portability and Accountability Act (HIPAA) standards based on X12 EDI Connector standards and version `005010`.
+Refer to this information for using X12 EDI Connector for HIPAA transaction sets, and for how usage differs from the standard X12.
```

Comment:
Refer to which information?
Also, correct sentence structure/parallelism: "Refer to...for using...and for how usage differs"